_posts/blog/2014-12-30-stress-test-long-run.md

```diff
@@ -22,19 +22,19 @@ everything finished, the test was a success. The number of unique integers
 computed independently by MapReduce matched the number computed by Fluo. Below
 is the output of the stress test count from Fluo.
 
-     $ java -cp $STRESS_JAR io.fluo.stress.trie.Print $FLUO_PROPS
+     $ java -cp $STRESS_JAR stresso.trie.Print $FLUO_PROPS
```

Comment:
Some of the find/replace was a bit aggressive. I'll fix any other errors you guys find like this.